lib/mayaUsd/utils/utilFileSystem.cpp

```diff
@@ -174,6 +174,23 @@ std::pair<std::string, bool> UsdMayaUtilFileSystem::makePathRelativeTo(
     return std::make_pair(relativePath.generic_string(), true);
 }
 
+std::string UsdMayaUtilFileSystem::getPathRelativeToDirectory(
+    const std::string& fileName,
+    const std::string& relativeToDir)
+{
+    auto relativePathAndSuccess = makePathRelativeTo(fileName, relativeToDir);
+
+    if (!relativePathAndSuccess.second) {
+        TF_WARN(
+            "File name (%s) cannot be resolved as relative to its parent layer directory (%s), "
```

Comment:
This function is general, so I would think we should not mention "parent layer" in the error.